samples/loadedDiscovery.html

```diff
@@ -0,0 +1,138 @@
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+
+  To run this sample, set apiKey to your application's API key and clientId to
+  your application's OAuth 2.0 client ID. They can be generated at:
+    https://console.developers.google.com/apis/credentials?project=_
+  Then, add a JavaScript origin to the client that corresponds to the domain
+  where you will be running the script. Finally, activate the People API at:
+    https://console.developers.google.com/apis/library?project=_
+-->
+<!DOCTYPE html>
+<html>
+  <!-- Note: this sample assumes the Fetch API and Promise are defined. -->
+  <head>
+    <title>Say hello using the People API</title>
+    <meta charset='utf-8' />
+  </head>
+  <body>
+    <p>Say hello using the People API.</p>
+
+    <!--Add buttons to initiate auth sequence and sign out-->
+    <button id="authorize-button" style="display: none;">Authorize</button>
+    <button id="signout-button" style="display: none;">Sign Out</button>
+
+    <div id="content"></div>
+
+    <script type="text/javascript">
+      // Enter an API key from the Google API Console:
+      //   https://console.developers.google.com/apis/credentials
+      var apiKey = 'YOUR_API_KEY';
+
+      // Enter a client ID for a web application from the Google API Console:
+      //   https://console.developers.google.com/apis/credentials?project=_
+      // In your API Console project, add a JavaScript origin that corresponds
+      //   to the domain where you will be running the script.
+      var clientId = 'YOUR_WEB_CLIENT_ID.apps.googleusercontent.com';
+
+      // Enter one or more authorization scopes. Refer to the documentation for
+      // the API or https://developers.google.com/people/v1/how-tos/authorizing
+      // for details.
+      var scopes = 'profile';
+
+      var authorizeButton = document.getElementById('authorize-button');
+      var signoutButton = document.getElementById('signout-button');
+
+      // The discovery JSON object.
+      var peopleApiDiscovery;
+
+      function handleClientLoad() {
+        // Load the API client and auth2 library.
+        var loadGapiClient = new Promise(function(resolve, reject) {
+          gapi.load('client:auth2', resolve);
+        });
+
+        // Fetch the People API discovery served on local server. You can also
+        // pack the json as a string to avoid extra network request.
+        // After this promise is fulfilled, the peopleApiDiscovery variable
+        // will be set.
+        var fetchPeopleApiDiscovery = fetch('people/people_rest_v1.json').then(
```

Comment:
Please include the Discovery Service URL used to fetch the original copy of this JSON file.